bin/dry-run.rb

```diff
@@ -35,6 +35,15 @@
 
 # rubocop:disable Style/GlobalVars
 
+unless File.exist?("/etc/dependabot-core-dev")
```

Comment:
WDYT about checking if the user is `dependabot` instead?

```
[dependabot-core-dev] ~/dependabot-core $ whoami
dependabot
```

Reply:
Yeah, I like that, it saves the hassle of having to rebuild images